ClangSharp.PInvokeGenerator.Test/VarDeclarationTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+
+namespace ClangSharp.Test
+{
+    public sealed class VarDeclarationTest : PInvokeGeneratorTest
+    {
+        [Theory]
+        [InlineData("unsigned char", "byte")]
+        [InlineData("double", "double")]
+        [InlineData("short", "short")]
+        [InlineData("int", "int")]
+        [InlineData("long long", "long")]
+        [InlineData("char", "sbyte")]
+        [InlineData("float", "float")]
+        [InlineData("unsigned short", "ushort")]
+        [InlineData("unsigned int", "uint")]
+        [InlineData("unsigned long long", "ulong")]
+        public async Task BasicTest(string nativeType, string expectedManagedType)
+        {
+            var inputContents = $@"{nativeType} MyVariable;";
+
+            var expectedOutputContents = $@"namespace ClangSharp.Test
+{{
+    public static partial class Methods
+    {{
+        private const string libraryPath = ""ClangSharpPInvokeGenerator"";
+
+        // public static extern {expectedManagedType} MyVariable;
```

Comment:
As mentioned, ideally this would be an `IntPtr` or a `T*` (depending on if safe or unsafe code) and would use the native library loader APIs (only available in .NET Core 3 and higher) to initialize the handle/address correctly.

I'll likely fix this up in a future PR, but for now, it lets consumers more easily see when these variables exist and take action themselves.